Makefile

```diff
@@ -17,15 +17,15 @@ else
     VECHO = @printf
 endif
 
-OBJS := kvm-host.o
+OBJS := kvm-host.o vm.o
```

Comment:
Move `vm.o` before `kvm-host.o` so that `#if` guard can take place before generic `kvm-host`.